oak_functions/examples/hello_world/module/src/tests.rs

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use hyper::client::Client;
+use oak_functions_loader::server::create_and_start_server;
+
+const WASM_MODULE_NAME: &str = "hello_world.wasm";
+const EXPECTED_RESPONSE: &str = "Welcome to Oak functions!\nRequest length: 0.\n";
+
+#[tokio::test]
+async fn test_server() {
+    let _ = env_logger::builder().is_test(true).try_init();
+    let port = 9999;
+    let address = format!("[::]:{}", port);
```

Comment:
See https://github.com/project-oak/oak/pull/1960/files for a nicer way of expressing special addresses (like `[::]` and `[::1]`).